src/github/queries.gql

```diff
@@ -270,6 +290,63 @@ query PullRequest($owner: String!, $name: String!, $number: Int!) {
 	}

```

Comment:
I forgot to mention this - there's also an `enterprise.gql` file that is nearly identical to this. This is meant to track the schema of the oldest supported version of GitHub enterprise, which in some cases is missing properties compared to this file. Even though I don't have a way of testing enterprise, I'd prefer to keep it (theoretically) working. Can you copy these changes there?

Reply:
Copied!